.github/workflows/e2e.yml

```diff
@@ -11,7 +11,31 @@ concurrency:
 
 jobs:
   systems-test:
-    uses: stellar/system-test/.github/workflows/test.yml@master
+   # if: github.event_name == 'push'
```

Comment:
should this be uncommented or removed?

Reply:
Yeah it was commented-out for testing purposes 👍🏼